THIRD-PARTY.md

```diff
@@ -44,6 +44,9 @@
 #### map_3d
 - [Source code](https://github.com/gberrante/map_3d)
 
+#### MQTT NIO
+- [Source code](https://github.com/swift-server-community/mqtt-nio)
+
```

Comment:
Beware, the references in this file are sorted by language, and here you are in the Rust ones  
You should add a Swift section for this

```md
## Swift

### License Apache 2.0

#### MQTT NIO
- [Source code](https://github.com/swift-server-community/mqtt-nio)
```

Reply:
Thanks, I miss the language section , I thought it was only classified by license.